recipe/meta.yaml

```diff
@@ -8,14 +8,17 @@ source:
   fn: gsl-{{ version }}.tar.gz
   url: http://ftpmirror.gnu.org/gsl/gsl-{{ version }}.tar.gz
   sha256: 13d23dc7b0824e1405f3f7e7d0776deee9b8f62c62860bf66e7852d402b8b024
+  
+  patches:                         # [win]
+  # https://github.com/ampl/gsl/  
+    - patch.diff                   # [win]
```

Comment:
Do you need the patches directory?
Can we get a nice name and comment describing the patch you are applying here?